index.css

```diff
@@ -0,0 +1,93 @@
+*{
+    border-color: rgba(0,0,0,0);
```

Comment:
One 🐼 died. Only `button`s have border by default, why removing non-existing things for elsewhere?

Reply:
I later on understood why you wrote this here. It is okay now, but avoid doing this in the future.